src/Compilers/VisualBasic/Portable/Symbols/ArrayTypeSymbol.vb

```diff
@@ -317,6 +317,41 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return IsSameType(obj, TypeCompareKind.ConsiderEverything)
         End Function
 
+        Friend Overrides Function Equals(t2 As TypeSymbol, comparison As TypeCompareKind) As Boolean
```

Comment:
>Equals [](start = 34, length = 6)

Each of these implementations could use some unit tests. Presumably there are some C# tests that can be ported.